Synonyms route changes and wording (#111)
meilisearch/documentation: advanced_guides/synonyms.md

To make your search more relevant there is the possibility of creating synonyms.

> Thus, if your database contains a lot of a term that has on or more popular synonyms, by creating a synonym list, terms can be associated. **This allows the same search results if you query one or the other terms**.

Reviewer: ```suggestion
Your database might contain a lot of a term that has one or more popular synonyms. By creating a synonym list, these terms can be associated. **This allows the same search results if you query one or the other terms**.
```

Author: Nice suggestion

There are several ways to associate words with each other.

> ## A is synonym of B but B is not synonym of A

Reviewer: ```suggestion
## A is a synonym for B but B is not a synonym for A
```

Reviewer: https://www.google.com/search?client=firefox-b-d&sxsrf=ACYBGNTywDmBX9Uj4uO29cIg6aNF6oIyqA%3A1578486234863&ei=2skVXoisNJDkU6PFk4AN&q=what+is+a+synonym&oq=what+is+a+synonym&gs_l=psy-ab.3..0i203l10.3474.5676..5982...1.0..0.130.1341.18j2......0....1..gws-wiz.......35i39j35i39i19j0j0i67j0i131.vncM7uml6L4&ved=0ahUKEwiIgb-6__PmAhUQ8hQKHaPiBNAQ4dUDCAo&uact=5

Reviewer: https://www.usingenglish.com/forum/threads/162330-synonym-of-for

Reviewer: I haven't check all along with the doc, but there are certainly some sentences with this mistake.

By searching `phone` you will get all results containing `iphone` with the same relevance. However, if you search for `iphone`, the data containing `phone` will not change your results.

> ## A is synonym of B and B is synonym of A

Reviewer: ```suggestion
## A is a synonym for B and B is a synonym of A
```
